internal/runnerinstall/ecs.go

```diff
@@ -359,89 +356,110 @@ func (i *ECSRunnerInstaller) Uninstall(ctx context.Context, opts *InstallOpts) e
 			return err
 		}
 
-		s.Update("Runner uninstalled")
+		s.Update("Waypoint runner AWS ECS service deleted")
 	}
 	s.Done()
 
 	// TODO: Still attempt to delete the EFS volume if the ECS service
 	// uninstall fails
+	s = sg.Add("Deleting runner file system")
 	efsSvc := efs.New(sess)
 	marker := ""
 	done := false
 	var fileSystems []*efs.FileSystemDescription
 	for !done {
-		fileSystemsResp, err := efsSvc.DescribeFileSystems(&efs.DescribeFileSystemsInput{
-			Marker:   aws.String(marker),
+		req := &efs.DescribeFileSystemsInput{
```

Comment:
We could possibly reduce this function by using [DescribeFileSystemsPages](https://docs.aws.amazon.com/sdk-for-go/api/service/efs/#EFS.DescribeFileSystemsPages) to handle the pagination logic for us, so something like this:

```go
var fileSystems []*efs.FileSystemDescription
err := client.DescribeFileSystemsPages(
  &efs.DescribeFileSystemsInput{}, // don't need to set marker. MaxItems also defaults to 100
  func(page *efs.DescribeFileSystemsOutput, lastPage bool) bool {
		fileSystems = append(fileSystems, page.FileSystems...)
		
    // double check that lastPage is the API telling us it's the last page though
    return lastPage 
})
```